### PR TITLE
Migration tests from JUnit4 to JUnit5 - first iteration

### PR DESCRIPTION
**Pull request by Captain1653**
It will be a really big task. It's the first iteration. Should I create some separate issue or not?

Kind regards

### CODE REVIEW

**Comment by Captain1653** [comment]
Hm... some strange situation. All my changes are related to `cloudfondry-indentity-server`. I run `./gradlew :cloudfoundry-identity-server:test` and my tests are green. 

`./run-unit-tests.sh hsqld` became to crash Gradle Daemon. Did I miss something?

File: server/src/test/java/org/cloudfoundry/identity/uaa/account/ResetPasswordAuthenticationEntryPointTests.java [hunk 1]
```diff
@@ -40,28 +40,25 @@
...
+    void setup() {
         request = mock(HttpServletRequest.class);
         response = mock(HttpServletResponse.class);
         requestDispatcher = mock(RequestDispatcher.class);
 
         email = "test@test.org";
-        code = "12345";
-        password = "mypassword";
-        passwordConfirmation = "mypassword";
+        String code = "12345";
```
Comment (strehle) [inline server/src/test/java/org/cloudfoundry/identity/uaa/account/ResetPasswordAuthenticationEntryPointTests.java]: in java 11 you can omit this

File: server/src/test/java/org/cloudfoundry/identity/uaa/account/ResetPasswordAuthenticationFilterTest.java [hunk 2]
```diff
@@ -68,21 +66,20 @@
...
     }
 
-    @Before
-    public void setup() {
-        codeStore = new InMemoryExpiringCodeStore(new TimeServiceImpl());
-        code = codeStore.generateCode("{}", new Timestamp(System.currentTimeMillis() + 10*60*1000), "", IdentityZoneHolder.get().getId()).getCode();
+    @BeforeEach
+    void setup() {
+        InMemoryExpiringCodeStore codeStore = new InMemoryExpiringCodeStore(new TimeServiceImpl());
+        String code = codeStore.generateCode("{}", new Timestamp(System.currentTimeMillis() + 10 * 60 * 1000), "", IdentityZoneHolder.get().getId()).getCode();
```
Comment (strehle) [inline server/src/test/java/org/cloudfoundry/identity/uaa/account/ResetPasswordAuthenticationFilterTest.java]: in java 11 you can omit this

**Review by strehle — COMMENTED** [review]
so far ok, thanks,

**Review by strehle — APPROVED** [review]
LGTM

**Pull request merged**